[2025-04-28T00-39-40-430Z] ERROR:
Name: Error
Message: UNHANDLED_REJECTION
Stack:
Error: UNHANDLED_REJECTION
    at Object.error (/home/container/events/logger.js:152:27)
    at process.<anonymous> (/home/container/index.js:210:10)
    at process.emit (node:events:518:28)
    at emitUnhandledRejection (node:internal/process/promises:252:13)
    at throwUnhandledRejectionsMode (node:internal/process/promises:388:19)
    at processPromiseRejections (node:internal/process/promises:475:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:106:32)
Metadata: {
  "requestBody": {
    "files": [],
    "json": {
      "type": 4,
      "data": {
        "content": "Error al procesar la interacci√≥n",
        "tts": false,
        "enforce_nonce": false,
        "flags": 64
      }
    }
  },
  "rawError": {
    "message": "Unknown interaction",
    "code": 10062
  },
  "code": 10062,
  "status": 404,
  "method": "POST",
  "url": "https://discord.com/api/v10/interactions/1366212236418220193/aW50ZXJhY3Rpb246MTM2NjIxMjIzNjQxODIyMDE5Mzp4Y25BUTlqeHRhTjFaUXhvdFRRZVdVdzlyN1FudDNwd09CeWtJaFJ4eEhtSUk4NGx0MTQ2VmxVSGRqOGY3ZkdaYkV4NmhaYTFLdWJ6WUY3R3k1bk40WlFHcTF1QUhIT0taOTM5V25yQ29nWmxpdk1vZDFBQXlEeTQ1Z0xNOVZraQ/callback?with_response=false"
}

